src/qt/forms/overviewpage.ui

```diff
@@ -168,14 +168,40 @@
               </property>
              </widget>
             </item>
-            <item row="4" column="0" colspan="2">
+            <item row="4" column="0">
+             <widget class="QLabel" name="labelUsedText">
+              <property name="text">
+               <string>Used:</string>
+              </property>
+             </widget>
+            </item>
+            <item row="4" column="1">
+             <widget class="QLabel" name="labelUsed">
+              <property name="cursor">
+               <cursorShape>IBeamCursor</cursorShape>
+              </property>
+              <property name="toolTip">
+               <string>Your current used balance</string>
```

Comment:
> > Because the space is tiny, and the impact should be big, every word in an effective tooltip needs to pack a punch.
> 
> Source: Internet

as the tooltip for the "used" line of the "balances" block,
"your current used balance" doesnt feel like a great tooltip